apps/codeforafrica/src/payload/globals/Publication/PostsTab.js

```diff
@@ -0,0 +1,82 @@
+const PostsTab = {
+  label: "Posts",
```

Comment:
```suggestion
  label: "Post",
```

Since this is for individual posts, may be we should name it so.